export __VMALLOC_RESERVE kernel symbol for Nvidia x86 Linux driver 1.0-6111
taken from http://kerneltrap.org/node/view/4022

--- linux-orig/arch/i386/mm/init.c      2004-09-26 03:43:57.944613000 +1000
+++ linux-new/arch/i386/mm/init.c       2004-09-28 02:37:21.787922000 +1000
@@ -41,6 +41,7 @@
 #include <asm/sections.h>

 unsigned int __VMALLOC_RESERVE = 128 << 20;
+EXPORT_SYMBOL(__VMALLOC_RESERVE);

 DEFINE_PER_CPU(struct mmu_gather, mmu_gathers);
 unsigned long highstart_pfn, highend_pfn;
 